<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
    <title>HPC tools</title>
    <meta name="description" content="High Performance Computing tools project page" />

    <link rel="stylesheet" href="css/style.css" />

    <link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <link
      href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600;700;900&display=swap"
      rel="stylesheet"
    />
  </head>
  <body>
    <header class="header">
      <div class="header__content">
        <div class="header__logo-container">
          <div class="header__logo-img-cont">
            <img
              src="./assets/png/semillero.png"
              alt="Ram Maheshwari Logo Image"
              class="header__logo-img"
            />
          </div>
          <span class="header__logo-sub">Team Bioinformatics and Artificial Intelligence</span>
        </div>
        <div class="header__main">
          <ul class="header__links">
            <li class="header__link-wrapper">
              <a href="./index.html" class="header__link"> Home </a>
            </li>
            <li class="header__link-wrapper">
              <a href="./index.html#about" class="header__link">About </a>
            </li>
            <li class="header__link-wrapper">
              <a href="./index.html#projects" class="header__link">
                Projects
              </a>
            </li>
            <!--
            <li class="header__link-wrapper">
              <a href="./index.html#contact" class="header__link"> Contact </a>
            </li>
            -->
          </ul>
          <div class="header__main-ham-menu-cont">
            <img
              src="./assets/svg/ham-menu.svg"
              alt="hamburger menu"
              class="header__main-ham-menu"
            />
            <img
              src="./assets/svg/ham-menu-close.svg"
              alt="hamburger menu close"
              class="header__main-ham-menu-close d-none"
            />
          </div>
        </div>
      </div>
      <div class="header__sm-menu">
        <div class="header__sm-menu-content">
          <ul class="header__sm-menu-links">
            <li class="header__sm-menu-link">
              <a href="./index.html"> Home </a>
            </li>

            <li class="header__sm-menu-link">
              <a href="./index.html#about"> About </a>
            </li>

            <li class="header__sm-menu-link">
              <a href="./index.html#projects"> Projects </a>
            </li>

            <li class="header__sm-menu-link">
              <a href="./index.html#contact"> Contact </a>
            </li>
          </ul>
        </div>
      </div>
    </header>
    



    <section class="project-cs-hero">
      <div class="project-cs-hero__content">
        <h1 class="heading-primary">
          Parallel Programming in Biological Sciences</h1>
        <div class="project-cs-hero__info">
          <p class="text-primary">
            Taking Advantage of Supercomputing in Genomics
          </p>
        </div>
      </div>
    </section>
    <section class="project-details">
      <div class="main-container">
        <div class="project-details__content">
          <div class="project-details__showcase-img-cont">
            <img
              src="./assets/projects/supercomp.png"
              alt="Project Image"
              class="project-details__showcase-img"
            />
          </div>
          <div class="project-details__content-main">
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Project Overview</h3>
              <p class="project-details__desc-para">
                New sequencing technologies has been increasing the size of current genomes 
                rapidly reducing its cost at the same time, those data need to be processed with 
                efficient and innovated tools using high performance computing (HPC), but for 
                taking advantage of nowadays supercomputers, parallel programming techniques 
                and strategies have to be used.
              </p>
              
            </div>
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Results Highlight</h3>
              <p class="project-details__desc-para">
                Here we presented the utility of multi-core architectures and parallel programming 
                for analyzing and classifying massive quantity of genomic information up to 16 times faster. 
              </p>         
            </div>
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Products</h3>
              <p class="project-details__desc-para">
                [1] Orozco-Arias, S., Tabares-Soto, R., Ceballos, D., Guyot, R. (2017). Parallel Programming in Biological Sciences, 
                Taking Advantage of Supercomputing in Genomics. In: Solano, A., Ordo√±ez, H. (eds) Advances in Computing. 
                CCC 2017. <i>Communications in Computer and Information Science, vol 735. Springer, cham </i>
                <a style="color: #0062b9;  text-decoration-line: underline;" href="https://doi.org/10.1007/978-3-319-66562-7_45"
                target="_blank" rel="noreferrer">  https://doi.org/10.1007/978-3-319-66562-7_45 </a>
              </p>
            </div>
            <div class="project-details__tools-used">
              <h3 class="project-details__content-title">Tools Used</h3>
              <div class="skills">
                <div class="skills__skill">Bash</div>
                <div class="skills__skill">Linux</div>
                <div class="skills__skill">MATLAB</div>
              </div>
            </div>
            <div class="project-details__links">
              <h3 class="project-details__content-title">See Resources</h3>
              <a
                href="https://link.springer.com/chapter/10.1007/978-3-319-66562-7_45#"
                class="btn btn--med btn--theme project-details__links-btn"
                target="_blank"
                >Paper Link</a
              >
            </div>
          </div>
        </div>
      </div>
    </section>




    <section class="project-cs-hero">
      <div class="project-cs-hero__content">
        <h1 class="heading-primary">Inpactor</h1>
        <div class="project-cs-hero__info">
          <p class="text-primary">
            Integrated and Parallel Analyzer and Classifier of LTR Retrotransposons 
            and Its Application for Pineapple LTR Retrotransposons Diversity and Dynamics 
          </p>
        </div>
      </div>
    </section>
    <section class="project-details">
      <div class="main-container">
        <div class="project-details__content">
          <div class="project-details__showcase-img-cont">
            <img
              src="./assets/projects/inpactor.jpg"
              alt="Project Image"
              class="project-details__showcase-img"
            />
          </div>
          <div class="project-details__content-main">
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Project Overview</h3>
              <p class="project-details__desc-para">
                One particular class of Transposable Elements (TEs), called Long Terminal Repeats (LTRs), 
                retrotransposons, comprises the most abundant mobile elements in plant genomes. Their 
                copy number can vary from several hundreds to up to a few million copies per genome, 
                deeply affecting genome organization and function. The detailed classification of 
                LTR retrotransposons is an essential step to precisely understand their effect at the 
                genome level, but remains challenging in large-sized genomes, requiring the use of 
                optimized bioinformatics tools that can take advantage of supercomputers.
              </p>
              
            </div>
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Results Highlight</h3>
              <p class="project-details__desc-para">
                The pineapple genome assembly comprises 44% of transposable elements, of which 23% were 
                classified as LTR retrotransposons. Exceptionally, 16.4% of the pineapple genome assembly 
                corresponded to only one lineage of the Gypsy superfamily: Del, suggesting that this 
                particular lineage has undergone a significant increase in its copy numbers. 
                As demonstrated for the pineapple genome, Inpactor provides comprehensive data of LTR 
                retrotransposons' classification and dynamics, allowing a fine understanding of their 
                contribution to genome structure and evolution. 
              </p>         
            </div>
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Products</h3>
              <p class="project-details__desc-para">
                [1] Orozco-Arias S, Liu J, Tabares-Soto R, Ceballos D, Silva Domingues D, Garavito A, Ming R, 
                Guyot R. Inpactor, Integrated and Parallel Analyzer and Classifier of LTR Retrotransposons and 
                Its Application for Pineapple LTR Retrotransposons Diversity and Dynamics. Biology (Basel). 2018 May 25;7(2):32. 
                <a style="color: #0062b9;  text-decoration-line: underline;" href="10.3390/biology7020032"
                target="_blank" rel="noreferrer">  doi: 10.3390/biology7020032</a>.PMID: 29799487; PMCID: PMC6022998.
              </p>
            </div>
            <div class="project-details__tools-used">
              <h3 class="project-details__content-title">Tools Used</h3>
              <div class="skills">
                <div class="skills__skill">Bash</div>
                <div class="skills__skill">Linux</div>
                <div class="skills__skill">multiprocess</div>
                <div class="skills__skill">Ltr-finder</div>
              </div>
            </div>
            <div class="project-details__links">
              <h3 class="project-details__content-title">See Resources</h3>
              <a
                href="https://www.mdpi.com/2079-7737/7/2/32"
                class="btn btn--med btn--theme project-details__links-btn"
                target="_blank"
                >Paper Link</a
              >
              <a
                href="https://github.com/simonorozcoarias/Inpactor"
                class="btn btn--med btn--theme-inv project-details__links-btn"
                target="_blank"
                >Code Link</a
              >
            </div>
          </div>
        </div>
      </div>
    </section>


    <section class="project-cs-hero">
      <div class="project-cs-hero__content">
        <h1 class="heading-primary">TIP_finder</h1>
        <div class="project-cs-hero__info">
          <p class="text-primary">
            An HPC software to detect transposable element insertion polymorphism in large genomic datasets
          </p>
        </div>
      </div>
    </section>
    <section class="project-details">
      <div class="main-container">
        <div class="project-details__content">
          <div class="project-details__showcase-img-cont">
            <img
              src="./assets/projects/TIPfinder.jpg"
              alt="Project Image"
              class="project-details__showcase-img"
            />
          </div>
          <div class="project-details__content-main">
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Project Overview</h3>
              <p class="project-details__desc-para">
                Transposable elements (TEs) are non-static genomic units capable of moving indistinctly 
                from one chromosomal location to another. Their insertion polymorphisms may cause beneficial 
                mutations, such as the creation of new gene function, or deleterious in eukaryotes, e.g., 
                different types of cancer in humans. A particular type of TE called LTR-retrotransposons 
                comprises almost 8% of the human genome. Among LTR retrotransposons, human endogenous 
                retroviruses (HERVs) bear structural and functional similarities to retroviruses. 
                Several tools allow the detection of transposon insertion polymorphisms (TIPs) but fail 
                to efficiently analyze large genomes or large datasets. 
              </p>
              
            </div>
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Results Highlight</h3>
              <p class="project-details__desc-para">
                Here, we developed a computational tool, named TIP_finder, able to detect mobile 
                element insertions in very large genomes, through high-performance computing (HPC) 
                and parallel programming, using the inference of discordant read pair analysis. 
                TIP_finder accelerates the detection of transposon insertion polymorphisms (TIPs) 
                by up to 55 times in breast cancer datasets and 46 times in cancer-free datasets 
                compared to the fastest available algorithms. TIP_finder applies a validated 
                strategy to find TIPs, accelerates the process through HPC, and addresses the 
                issues of runtime for large-scale analyses in the post-genomic era.
              </p>         
            </div>
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Products</h3>
              <p class="project-details__desc-para">
                [1] Orozco-Arias, S.; Tobon-Orozco, N.; Pi√±a, J.S.; Jim√©nez-Var√≥n, C.F.; Tabares-Soto, R.; Guyot, R. TIP_finder: An HPC Software to Detect Transposable Element Insertion Polymorphisms in Large Genomic Datasets. Biology 2020, 9, 281.
                <a style="color: #0062b9;  text-decoration-line: underline;" href=" 10.3390/biology9090281 "
                target="_blank" rel="noreferrer">  doi:  10.3390/biology9090281 </a>.
              </p>
            </div>
            <div class="project-details__tools-used">
              <h3 class="project-details__content-title">Tools Used</h3>
              <div class="skills">
                <div class="skills__skill">Python</div>
                <div class="skills__skill">Linux</div>
                <div class="skills__skill">Bioinformatic Softwares</div>
                <div class="skills__skill">mpi4py</div>
                <div class="skills__skill">Anaconda</div>

              </div>
            </div>
            <div class="project-details__links">
              <h3 class="project-details__content-title">See Resources</h3>
              <a
                href="https://pubmed.ncbi.nlm.nih.gov/32917036/"
                class="btn btn--med btn--theme project-details__links-btn"
                target="_blank"
                >Paper Link</a
              >
              <a
                href="https://github.com/simonorozcoarias/TIP_finder"
                class="btn btn--med btn--theme-inv project-details__links-btn"
                target="_blank"
                >Code Link</a
              >
            </div>
          </div>
        </div>
      </div>
    </section>






    <section class="project-cs-hero">
      <div class="project-cs-hero__content">
        <h1 class="heading-primary">G-SAIP</h1>
        <div class="project-cs-hero__info">
          <p class="text-primary">
            Graphical Sequence Alignment Through Parallel Programming in the Post-Genomic Era 
          </p>
        </div>
      </div>
    </section>
    <section class="project-details">
      <div class="main-container">
        <div class="project-details__content">
          <div class="project-details__showcase-img-cont">
            <img
              src="./assets/projects/gsaip.jpg"
              alt="Project Image"
              class="project-details__showcase-img"
            />
          </div>
          <div class="project-details__content-main">
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Project Overview</h3>
              <p class="project-details__desc-para">
                A common task in bioinformatics is to compare DNA sequences to identify 
                similarities between organisms at the sequence level. An approach to such 
                comparison is the dot-plots, a 2-dimensional graphical representation to 
                analyze DNA or protein alignments. Dot-plots alignment software existed 
                before the sequencing revolution, and now there is an ongoing limitation 
                when dealing with large-size sequences, resulting in very long execution times.
              </p>
              
            </div>
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Results Highlight</h3>
              <p class="project-details__desc-para">
                we present G-SAIP (Graphical Sequence Alignment in Parallel), a software 
                capable of spawning multiple distributed processes on CPUs, over a 
                supercomputing infrastructure to speed up the execution time for dot-plot 
                generation up to 1.68x compared with other current fastest tools, improve 
                the efficiency for comparative structural genomic analysis, phylogenetics 
                because the benefits of pairwise alignments for comparison between genomes, 
                repetitive structure identification, and assembly quality checking. 
              </p>         
            </div>
            <div class="project-details__desc">
              <h3 class="project-details__content-title">Products</h3>
              <p class="project-details__desc-para">
                [1] Pi√±a JS, Orozco-Arias S, Tob√≥n-Orozco N, Camargo-Forero L, Tabares-Soto R, Guyot R. 
                G-SAIP: Graphical Sequence Alignment Through Parallel Programming in the Post-Genomic Era. 
                Evol Bioinform Online. 2023 Jan 20;19:11769343221150585.
                <a style="color: #0062b9;  text-decoration-line: underline;" href="10.1177/11769343221150585"
                target="_blank" rel="noreferrer">  doi: 10.1177/11769343221150585 </a>.
              </p>
            </div>
            <div class="project-details__tools-used">
              <h3 class="project-details__content-title">Tools Used</h3>
              <div class="skills">
                <div class="skills__skill">Python</div>
                <div class="skills__skill">mpi4py</div>
                <div class="skills__skill">Bioinformatic Softwares</div>
                <div class="skills__skill">multiprocess</div>
                <div class="skills__skill">Anaconda</div>

              </div>
            </div>
            <div class="project-details__links">
              <h3 class="project-details__content-title">See Resources</h3>
              <a
                href="https://pubmed.ncbi.nlm.nih.gov/36703866/"
                class="btn btn--med btn--theme project-details__links-btn"
                target="_blank"
                >Paper Link</a
              >
              <a
                href="https://github.com/simonorozcoarias/G-SAIP"
                class="btn btn--med btn--theme-inv project-details__links-btn"
                target="_blank"
                >Code Link</a
              >
            </div>
          </div>
        </div>
      </div>
    </section>





    <footer class="main-footer">
      <div class="main-container">
        <div class="main-footer__upper">
          <div class="main-footer__row main-footer__row-1">
            <h2 class="heading heading-sm main-footer__heading-sm">
              <span>Social</span>
            </h2>
            <div class="main-footer__social-cont">
              <a target="_blank" rel="noreferrer" href="https://github.com/BioAITeam">
                <img
                  class="main-footer__icon"
                  src="./assets/png/github-ico.png"
                  alt="icon"
                />
              </a>
              <!--
              <a target="_blank" rel="noreferrer" href="#">
                <img
                  class="main-footer__icon"
                  src="./assets/png/linkedin-ico.png"
                  alt="icon"
                />
              </a>
              <a target="_blank" rel="noreferrer" href="#">
                <img
                  class="main-footer__icon"
                  src="./assets/png/twitter-ico.png"
                  alt="icon"
                />
              </a>
              <a target="_blank" rel="noreferrer" href="#">
                <img
                  class="main-footer__icon"
                  src="./assets/png/yt-ico.png"
                  alt="icon"
                />
              </a>
              <a target="_blank" rel="noreferrer" href="#">
                <img
                  class="main-footer__icon main-footer__icon--mr-none"
                  src="./assets/png/insta-ico.png"
                  alt="icon"
                />
              </a>
              -->
            </div>
          </div>
          <div class="main-footer__row main-footer__row-2">
            <h4 class="heading heading-sm text-lt">Team Bioinformatics and Artificial Intelligence</h4>
            <p class="main-footer__short-desc">
              <!-- Universidad Aut√≥noma de Manizales </br> Contact at: <a href="mailto:rtabares@autonoma.edu.co">rtabares@autonoma.edu.co</a> -->
            </p>
          </div>
        </div>

        <!-- If you give me some credit or shoutout here by linking to my website, then it will be a big thing for me and will help me a lot :) -->
        <div class="main-footer__lower">
          &copy; Copyright 2021. Made by
          <a rel="noreferrer" target="_blank" href="https://rammaheshwari.com"
            >Ram Maheshwari</a
          >
        </div>
      </div>
    </footer>
    <script src="./index.js"></script>
  </body>
</html>
